### DISCUSSION TITLE
API Token having multiple scopes

### DISCUSSION THREAD

**Discussion by zachtownes** [TandoorRecipes/recipes#3090] [Q&A] [answered]
Hey, I was trying to give an api token read, write permissions. I wasn't able to find the right combination of read/write together to get permissions for both Get and Put requests.

**Comment by smilerz** [comment, marked as answer]
'read write'

**Reply by zachtownes** [reply]
Thank you! I feel dumb for not trying that.... but Thanks for the help!
